### PR TITLE
boards: Remove all deprecated HWMv1 board aliases

### PR DESCRIPTION
**Pull request by aescolar**
They have been there for 2 releases. Users must now use the HWMv2 names

### CODE REVIEW

File: boards/deprecated.cmake [hunk 1]
```diff
@@ -13,903 +13,9 @@
...
-)
-set(actinius_icarus_som_dk_ns_DEPRECATED
-    actinius_icarus_som_dk/nrf9160/ns
-)
-set(actinius_icarus_som_ns_DEPRECATED
-    actinius_icarus_som/nrf9160/ns
-)
-set(adafruit_feather_DEPRECATED
-    adafruit_feather_nrf52840/nrf52840
-)
```
Comment (57300) [inline boards/deprecated.cmake:39]: Should we keep this one? It was added after v3.7.0 (in 5e789c3ae0c7cac997d03ab73b778a334e5186a0 as a follow-up to 4e830ec19f5e7475b673816fc092f0c2a9352a3e)

**Review by kartben — CHANGES_REQUESTED** [review]
Similar to what @57300 said -- this touches too much it seems, see e.g 4dc2f6fd1cacb8cda3a658ccafff2ba00e624b9e or 2ec5fb4a4b8c11d7f4a82c85d2033b068cc06152, these shouldn't be removed, and prob others

File: boards/deprecated.cmake (hunk 1 above)
Comment (aescolar) [reply boards/deprecated.cmake]: Added back :)

**Comment by aescolar** [comment]
> Similar to what @57300 said -- this touches too much it seems, see e.g [4dc2f6f](https://github.com/zephyrproject-rtos/zephyr/commit/4dc2f6fd1cacb8cda3a658ccafff2ba00e624b9e) or [2ec5fb4](https://github.com/zephyrproject-rtos/zephyr/commit/2ec5fb4a4b8c11d7f4a82c85d2033b068cc06152), these shouldn't be removed, and prob others

Would you be so kind as to be more specific about what is too much? (I did check all boards but I may have made some mistake). wrt. the 2 commits you pointed the mimx8mm_phyboard_polis/mimx8mm6/m4 and mimx8mp_phyboard_pollux/mimx8ml8/m7 aliases are still there. What has been removed from those commits is the old pre 3.7.0 alias which had been changed to point to the new board name.

**Review by 57300 — APPROVED** [review]
LGTM. I used this Bash script to double check which aliases should stay:
```sh
git annotate boards/deprecated.cmake | while read -r line
do
	[[ "$line" =~ (^[0-9a-f]+) ]] && \
	git merge-base --is-ancestor v3.7.0 ${BASH_REMATCH[1]} && \
	echo "$line" | grep set
done
```

**Comment by kartben** [comment]
> What has been removed from those commits is the old pre 3.7.0 alias which had been changed to point to the new board name.

that's 100% correct. Sorry for the noise and my own confusion :)

**Review by kartben — APPROVED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]
Oh my, has it already been two releases. :smiley:

**Review by nordicjm — APPROVED** [review]

**Pull request merged**